tests/terraform/upgradecluster/upgradecluster_test.go

```diff
@@ -208,40 +243,50 @@ var _ = Describe("Upgrade Tests:", func() {
 	Context("Upgrade via SUC:", func() {
 		It("Verifies Upgrade", func() {
 			namespace := "system-upgrade"
-			_, err := terraform.DeployWorkload("suc.yaml", terraform.KubeConfigFile)
-			Expect(err).NotTo(HaveOccurred(), "system-upgrade-controller manifest did not deploy successfully")
+			_, err := terraform.DeployWorkload("suc.yaml", createcluster.KubeConfigFile)
+			Expect(err).NotTo(HaveOccurred(),
+				"system-upgrade-controller manifest did not deploy successfully")
 
 			Eventually(func(g Gomega) {
-				cmd := "kubectl get pods " + "-n " + namespace + " --kubeconfig=" + terraform.KubeConfigFile
+				cmd := "kubectl get pods " + "-n " + namespace + " --kubeconfig=" + createcluster.KubeConfigFile
 				res, _ := terraform.RunCommand(cmd)
+
 				g.Expect(res).Should(ContainSubstring("system-upgrade-controller"))
 				g.Expect(res).Should(ContainSubstring("Running"))
 			}, "120s", "2s").Should(Succeed())
 
-			err = upgradeCluster(*upgradeVersion, terraform.KubeConfigFile)
+			err = upgradeCluster(*upgradeVersion, createcluster.KubeConfigFile)
 			Expect(err).NotTo(HaveOccurred(), "failed to upgrade cluster.")
 
 			defer func() {
-				_, err := terraform.Nodes(terraform.KubeConfigFile, true)
+				_, err := terraform.Nodes(createcluster.KubeConfigFile, true)
 				if err != nil {
 					fmt.Println("Error retrieving nodes postupgrade: ", err)
 				}
-				_, err = terraform.Pods(terraform.KubeConfigFile, true)
+				_, err = terraform.Pods(createcluster.KubeConfigFile, true)
 				if err != nil {
 					fmt.Println("Error retrieving pods postupgrade: ", err)
 				}
 			}()
+
 			versionRegex := regexp.MustCompile("-rc[0-9]+")
 			k8sVersion := versionRegex.ReplaceAllString(*upgradeVersion, "")
+
 			fmt.Printf("\nFetching node status postupgrade\n")
-			expectedNodeCount := terraform.NumServers + terraform.NumWorkers
+
 			Eventually(func(g Gomega) {
-				nodes, err := terraform.Nodes(terraform.KubeConfigFile, false)
+				expectedNodeCount := createcluster.NumServers + createcluster.NumWorkers
```

Comment:
Similar to in createcluster, this variable isn't changing so probably belongs outside of the `Eventually` block.